jplotter/src/main/java/hageldave/jplotter/canvas/JPlotterCanvas.java

```diff
@@ -245,10 +245,13 @@ public default void paintPDF(PDDocument document, PDPage page) throws IOExceptio
 	}
 
 
-	public default void paintPDF(PDDocument document, PDPage page, PDPageContentStream contentStream, Rectangle2D renderLoc) throws IOException {
-		contentStream.addRect(0, 0, page.getMediaBox().getWidth(), page.getMediaBox().getHeight());
+	public default void paintPDF(PDDocument document, PDPage page, Rectangle2D renderLoc) throws IOException {
+		PDPageContentStream contentStream = new PDPageContentStream(document, page, PDPageContentStream.AppendMode.APPEND, false);
```

Comment:
I believe this is for drawing the background color of the component, correct? Maybe you can add an inline comment that says what this code does